src/main/webapp/data/userMapData.json

```diff
@@ -2009,18 +2009,18 @@
       "University of Washington",
       "University of West Florida",
       "University of Wisconsin",
-      "University of Wisconsin Oshkosh",
-      "University of Wisconsin School of Business",
-      "University of Wisconsin, Green Bay",
+	  "University of Wisconsin, Green Bay",
       "University of Wisconsin, La Crosse",
       "University of Wisconsin-Madison",
-      "University of Wisconsin-Platteville",
-      "Unniversity of Washington, Seattle",
+      "University of Wisconsin Oshkosh",
+	  "University of Wisconsin-Platteville",
+      "University of Wisconsin School of Business",
+      "University of Washington, Seattle",
```

Comment:
Uni of Washington Seattle should be right after the first Uni of Washington (line 2009)

Reply:
Yea, it should definitely be one 'n', if you can fix that